### ISSUE TITLE
Remove Sort option on 'Violation Record' column of Case Table

### ISSUE THREAD

**Issue by jovergaag** [Recidiviz/pulse-dashboard#554]
**Describe the bug**
Sorting the Case Table by violation record is extremely slow, and not helpful to the user because there is not a logical 'order' to these violation strings that have been concatenated together. The sort is slow because of the extreme uniqueness and variability of these strings, there is no avoiding this simply because of the nature of the data.

To avoid the user thinking that something is broken, simply remove the sortable option on that column.

@sychang Thoughts on this?

**To Reproduce**

**Expected behavior**

**Screenshots**
Screenshots revealing sensitive information _must not_ be shared. If a screenshot is helpful in diagnosing and resolving the bug, please send this directly to a contact on the Recidiviz team.

**Desktop (please complete the following information):**
 - OS [e.g. Windows 10]: 
 - Browser [e.g. chrome, edge]: 
 - Version [e.g. 22]: 

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]: 
 - OS: [e.g. iOS8.1]: 
 - Browser [e.g. stock browser, safari]: 
 - Version [e.g. 22]: 

**Additional context**

**Comment by sychang** [comment]
Yeah this makes sense to me. My only concern would be whether the user would perceive the lack of sort on the violation record column a bug not a feature. That said, I think it is fairly straightforward to see and understand that there is not a great ordering.

Context: Previously, we showed all violation records for an individual so people would often have multiple violations on a single violation record, making it slightly more interesting to sort this column. Since that was removed, I think it makes sense to also remove the sorting for this column. More details on past discussion here: https://github.com/Recidiviz/pulse-dashboard/issues/312.
